media/css/firefox/whatsnew/whatsnew-lite-in.scss

```diff
@@ -63,148 +63,153 @@
 //* -------------------------------------------------------------------------- */
 // Page content
 
-.lite-container {
-    display: flex;
-    justify-content: flex-end;
-    align-items: center;
-    flex-direction: column;
-    margin-top: $spacing-md;
+.lite-body {
+    background-color: #ade7e2;
+    background-image: url('/media/img/firefox/whatsnew/lite/background-in.jpg');
+    background-position: top;
+    color: $color-gray-80;
+    position: relative;
+    @include font-firefox;
```

Comment:
Not a blocker as usage here is debatable (the page copy is on the large side), but Metropolis is usually reserved for headings, and Inter for paragraph / body copy.

https://mozilla.design/firefox/typography/